modules/registration/registration_plugin.rb

```diff
@@ -0,0 +1,9 @@
+module Proxy::Registration
+  class Plugin < ::Proxy::Plugin
+    http_rackup_path  File.expand_path("http_config.ru", File.expand_path(__dir__))
+    https_rackup_path File.expand_path("http_config.ru", File.expand_path(__dir__))
+
+    plugin     :registration, ::Proxy::VERSION
+    capability :global_registration
```

Comment:
Now that it's a plugin, the capability doesn't make a lot of sense anymore. It's now promoted to a real Feature. Note that Foreman needs a PR as well to recognize the Feature.